src/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
+    <title>Leader Board</title>
+</head>
+<body>
+    <nav class="navbar navbar-expand-md navbar-light bg-light mt-5">
+        <h4 class="text-primary header">LEADERBOARD EX</h4>
+    </nav>
+
+    <div class="card container-sm p-5 appWrapper">
+    <div class="d-flex appContainer">
+        <div class="dynamicSec d-col">
+            <div class="d-flex"> <h3 id="heading">Recent Scores </h3>  <button id="refresh" class="btn btn-outline-primary">Refresh</button>  </div>
+            <div id="dynamicDisplay">
+
+            </div>
+        </div>
+        <div class="additionForm">
+            <form class="d-col">
+                <label for="name">Full Name</label>
+                <input type="text" name="name" id="name">
+                <label for="score">Your Score</label>
+                <input type="number" name="score" id="score">
+                <input type="button" value="Submit" class="btn btn-outline-success mt-2">
+                </form>
+        </div>
+    </div>
+</div>
+
```

Comment:
- [ ] Good job implementing the webpack :clap: However it would be nice to follow the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), provided by Microverse to follow the project requirement.
### Your design :point_down: 
![Screenshot from 2023-08-02 08-20-26](https://github.com/paulshaun-kdtk/leaderboard/assets/91142432/1a59a356-0b7b-4857-961c-e19cc5159963)